versioned_docs/version-8.1/components/concepts/process-instance-modification.md

```diff
@@ -8,7 +8,7 @@ Process instance modification is a powerful feature to repair a running process
 stuck on an element, waiting for an event, or taking an unintended path because an external system is not
 available or doesn't respond as expected, for example.
 
-Use the [modification command](/docs/apis-clients/grpc.md#modifyprocessinstance-rpc) to skip or repeat a step in the
+Use the [modification command](../../apis-clients/grpc.md#modifyprocessinstance-rpc) to skip or repeat a step in the
```

Comment:
non-blocking, note:

@christinaausley we changed this while pairing, and I hadn't pushed it up because I wasn't sure what the value should be. I'm confident now -- it should navigate relatively, and stay within the 8.1 version.